scripts/pod_lib_lint.rb

```diff
@@ -31,6 +31,7 @@ def usage()
   options can be any options for pod spec lint
 
   script options:
+    --skip-analyze: don't run Xcode analyze on this podspec
```

Comment:
Optional: conventionally inverted boolean argument flags are spelled with a no prefix (i.e. for `--analyze` the inverted form would conventionally be `--no-analyze`). Consider following this pattern here.

Reply:
Done, great idea.